src/actionspecials.h

```diff
@@ -236,4 +236,7 @@ DEFINE_SPECIAL(Ceiling_LowerToLowest, 253, 2, 2, 2)
 DEFINE_SPECIAL(Ceiling_LowerToFloor, 254, 2, 2, 2)
 DEFINE_SPECIAL(Ceiling_CrushRaiseAndStaySilA, 255, 4, 5, 5)
 
+DEFINE_SPECIAL(Line_SetVisualPortal, 155, -1, -1, 3)
+DEFINE_SPECIAL(Line_SetPortal, 156, -1, -1, 3)

```

Comment:
Nothing anymore. This is a relic of ancient past when Line_SetVisualPortal was 157 instead of 155. It's long fixed and this comment means nothing.